plugins/modeler

```diff
@@ -0,0 +1 @@
+repository = https://github.com/barmac/asdf-modeler.git
```

Comment:
this file should be named `camunda-modeler` as `modeler` is too generic for a single plugin to own.

Reply:
OK Done

Reply:
The `plugins/*` filename is what determines the plugin name in the plugin repo. I will fix